packages/core-compat-api/src/convertLegacyPageExtension.tsx

```diff
@@ -32,7 +32,11 @@ export function convertLegacyPageExtension(
   LegacyExtension: ComponentType<{}>,
   overrides?: {
     name?: string;
-    defaultPath?: string;
+    path?: string;
+    /**
+     * @deprecated Use the `path` param instead.
+     */
+    defaultPath?: [Error: `Use the 'path' override instead`];
```

Comment:
wait is that actually a label? :) that's a fascinating use - and it's pretty crazy it ends up in the api report too

Reply:
It's a labelled tuple element, like `[x: number, y: number]`